components/Sections/ImageTextColumns/ImageTextColumnsImage.tsx

```diff
@@ -0,0 +1,19 @@
+import { getFileInfo } from "@/lib/getAssetInfo";
+import { ImageComponent } from "../Shared/ImageComponent";
+
+export interface ImageTextColumnsImageBuilderProps {
+  id: string | null;
+}
+
+export async function ImageTextColumnsImageBuilder({
+  id,
+}: ImageTextColumnsImageBuilderProps) {
+  if (id === null) return null;
```

Comment:
Don’t check this here. Better check in the `ImageTextColumnsBuilder` if there is an image provided and only then use this component.